index.js

```diff
@@ -9,27 +9,10 @@ function ask(questionText) {
 
 start();
 
-function getRandomInt(max) {
-  return Math.floor(Math.random() * Math.floor(max));
-}
-
 async function start() {
   console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
   let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
   console.log('You entered: ' + secretNumber);
-
-let guess = getRandomInt(100) 
-let userResponse = await ask("Is " + guess + " your number?")
-
-
-while (check = false){let guess = getRandomInt(100) 
```

Comment:
this will set `check` to false causing the loop to never run. Single equals is for assignment, triple equals is for comparison.